ner_v1/detectors/textual/text/text_detection.py

```diff
@@ -154,20 +154,24 @@ def set_min_token_size_for_levenshtein(self, min_size):
         """
         self._min_token_size_for_fuzziness = min_size
 
-    def _process_text(self, text):
-        self.text = text.lower()
-        if isinstance(self.text, bytes):
-            self.text = self.text.decode('utf-8')
+    def _process_text(self, texts):
```

Comment:
Move this to a module level function or staticmethod which just takes in text and lowers and decodes if bytes. This function should still operate on one text at a time and should not assign anything to self.text. Do it outside where this function is called

Reply:
Done